docs/modules/integrate/pages/vector-collection-connector.adoc

```diff
@@ -0,0 +1,58 @@
+= Vector Collection Connector
+:description: Vector collection stores vectors with their related metadata. This allows entries to be found efficiently based on vector distance.
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+For further information on vector collections, see xref:data-structures:vector-collection.adoc[].
+
+== Installing the Connector
+
+This connector is included in the full and slim {enterprise-product-name} distributions of Hazelcast.
+
+== Permissions
+If xref:security:enabling-jaas.adoc[security] is enabled, you can set up permissions to restrict clients' access to these data structures.
+
+To search in vector collection, you must add the `create` and `read` permissions for those collections. If you use the vector collection sink to write to vector collections, you must add the `create` and `put` permissions for those collections.
+
+For further information on adding these permissions, see xref:security:native-client-security.adoc[].
```

Comment:
the link looks broken and does not resolve

Reply:
works for me in preview